config/webpack/base.config.js

```diff
@@ -51,11 +51,10 @@ module.exports = {
               {
                 loader: '@svgr/webpack',
                 options: {
-                  svgo: true,
                   svgoConfig: {
-                    full: true,
```

Comment:
`full` is true now by default (kinda): https://github.com/svg/svgo/releases/tag/v2.0.0

> In v1 full flag allowed to disable all default plugins and run only specified in plugins list. In v2 it's default behaviour.